### ISSUE TITLE
don't use so many colors that look the same

### ISSUE THREAD

**Issue by jrheard** [Yelp/firefly#43]
on this graph:

![graph](https://www.evernote.com/shard/s11/sh/e5da85a6-bfbb-4a7e-a712-3dd2c0f308e8/094cc761dbe65d49e70e95b2c2a57c30/res/d4db0b37-c7d0-41b5-a424-22d6aa726c6a/skitch.png)

the middle five sections are drawn using colors between yellowish-green and blueish-green, all of which look _extremely_ similar to my eye. when i see that a particular yellowish-greenish thing is disproportionately represented and i want to see which one it is, i have to spend five minutes squinting at the legend (not shown) and seeing which yellowish-green dot matches the section of the graph i'm interested in.

long story short - can you please make firefly not use so many colors that look the same? thanks!
